bus-mapping/src/evm/opcodes.rs

```diff
@@ -438,43 +434,39 @@ pub fn gen_end_tx_ops(
     )?;
 
     // handle tx receipt tag
-    state.tx_receipt_read(
+    state.tx_receipt_write(
```

Comment:
if change to write type, pls. update spec accordingly.

Reply:
can you take a look at https://github.com/privacy-scaling-explorations/zkevm-specs/pull/234 please? It won't let me add you as a reviewer.

Reply:
will have a look !

Reply:
approved the spec change.